psl/builtin-connectors/src/postgres_datamodel_connector/native_types.rs

```diff
@@ -0,0 +1,29 @@
+crate::definition! {
+    PostgresType;
+    SmallInt {Int},
```

Comment:
The variant in the curly braces is the Prisma type? We are using the enum in the macro I guess?

Reply:
yep that's the idea. I'll spend some time documenting the macro more.